src/BuildScriptGeneratorCli/BuildScriptGenerator.cs

```diff
@@ -47,7 +47,9 @@ public bool TryGenerateScript(out string generatedScript)
                     EnableNodeJs = !environment.DisableNodeJs,
                     EnablePython = !environment.DisablePython,
                     EnablePhp = !environment.DisablePhp,
-                    DisableMultiPlatformBuild = environment.DisableMultiPlatformBuild
+                    
+                    // Temporarily disable multi-platform builds
+                    DisableMultiPlatformBuild = true
```

Comment:
@gpcastro  As discussed earlier, I disabled mutli-platform build for now. Would this be the best place for it?